src/components/ContinuousField.tsx

```diff
@@ -1,21 +1,27 @@
 import TextField from '@mui/material/TextField';
 
-function ContinuousField({
-  label,
-  onFieldChange,
-}: {
+export interface ContinuousFieldProps {
+  error?: boolean;
+  helperText?: string;
   label: string;
   onFieldChange: (fieldLabel: string, value: string) => void;
-}) {
+}
+
+function ContinuousField({ error, helperText, label, onFieldChange }: ContinuousFieldProps) {
   return (
-    // TODO: see if we can make it so TextField returns type number instead of string as its doing now
     <TextField
```

Comment:
This is one of the things I came to after a couple of iterations but I think the main reason was it would allow text to be typed but it wouldn't handle it properly once the element is out of focus.

![Peek 2024-02-13 09-32](https://github.com/neurobagel/react-query-tool/assets/77515879/6858ffd9-4c11-428a-ab11-19ea6fffbce2)